src/app/product/[id]/page.tsx

```diff
@@ -0,0 +1,68 @@
+import { Metadata } from 'next'
+import Image from 'next/image'
+import React from 'react'
+import Counter from '../../../../components/Counter'
+import pic from '../../../img/Coffee.png'
+
+async function getItemById(id: string) {
+  const response = await fetch(`http://localhost:8083/api/v1/products/${id}`)
```

Comment:
May be it is better to store this variable `http://localhost:8083/api/v1/products/${id}` in constant?
if it is not clear what I am talking about, write me on Telegram

Reply:
fixed

Reply:
???

Reply:
If I understand you correctly, for now, it's not necessary to move the API calls to a separate file until they are connected. In the future, I believe we will move all the API requests to a separate file.